### DISCUSSION TITLE
Error Launching Prime Server in virtuel environment

### DISCUSSION THREAD

**Discussion by stenghe** [ansys/pyprimemesh#901] [Q&A] [answered]
Hi,

I used pyprime mesh for quite some time without any issues. Now I created a new Project with a virtuel environment. When I now want to launch pyprime, i obtain the following Error:

FileNotFoundError: No valid Ansys Prime Server found to launch.

In my old projects without a virtual environment, this problem does not occur. What is the reason for that and how can it be fixed?

I have Windows 11 and Ansys 2024 R1

Thanks a lot!

**Comment by waltersma** [comment, marked as answer]
Hi @stenghe

Please can you check which version of the client you have installed?  If you are using 24R1 server the client version you would need would be 0.5.X.  It could be that you have installed the latest available client that is for use with 24R2.

There is a one-to-one compatibility for the client server versions.  You can see a table of this information [here](https://prime.docs.pyansys.com/version/stable/getting_started/index.html).

**Comment by stenghe** [comment]
That was indeed the issue. Thanks

**Comment by stenghe** [comment]
After I installed the 0.5.1 Version of the client, Another Error occured, which didn't before:

Traceback (most recent call last):
  File "c:\Users\heinr\Documents\Masterthesis-local\UnitCellAnalyser\test.py", line 15, in <module>
    mesh_util.read(file_name=".\\Working_dir\\Fluid_model.scdocx")
  File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\lucid\mesh_util.py", line 92, in read
    prime.FileIO(self._model).import_cad(
  File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\core\fileio.py", line 504, in import_cad
    import_result = super().import_cad(temp_file_name, params)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\autogen\fileio.py", line 650, in import_cad
    result = self._comm.serve(self._model, command_name, self._object_id, args=args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\heinr\AppData\Roaming\Python\Python312\site-packages\ansys\meshing\prime\internals\error_handling.py", line 535, in wrapper_error_code
    raise PrimeRuntimeError(
ansys.meshing.prime.internals.error_handling.PrimeRuntimeError: CAD import failed.

I checked the Path of my CAD-File and also tried to import a step file. What could be the issue here?

**Reply by waltersma** [reply]
Hi @stenghe 

As this is a new question we should add as a new discussion https://github.com/ansys/pyprimemesh/discussions/904.
